source/help/getting-started/configuring-notifications.rst

```diff
@@ -0,0 +1,105 @@
+Configuring Notifications
+=========================
+
+Notifications in Mattermost alert you to unread messages and mentions.
+
+Unreads and Mentions
+----------------------------------------
+
+Unread Messages Indicator
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The name of a channel in the left-hand sidebar shows as **bold** when
+there are unread messages in the channel. Clicking on the channel
+removes the bold indicator, and brings you to the earliest unread
+message in the channel.
+
+You can choose not to show unread indicators in a channel by clicking
+**Channel Menu** > **Notification Preferences** > **Mark Channel
+Unread** and selecting **Only for mentions**. This will only notify you if a mention is triggered on your name or a
+keyword for which you're listening.
+
+Mentions Indicator
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Unread mentions or `your name and selected
```

Comment:
This sentence is a little confusing, are there any ways we could phrase it better? Also should the docs link be for the words "your name and selected keywords" or just "selected keywords"?